.clang-tidy

```diff
@@ -3,6 +3,8 @@ Checks: >
   clang-analyzer-*,
   modernize-concat-nested-namespaces,
   modernize-use-nullptr,
+  modernize-use-equals-default,
+  modernize-use-equals-delete,
```

Comment:
Can we make a point to keep these checks in alphabetical order? That means we sort the top half of the checks (the ones that add checks) and the bottom half (where certain ones are removed) independently of each other. If you sort all of them then the removals get placed before the additions which messes things up.

Reply:
OK, got it.